### ISSUE TITLE
 reproduce the baseline (ResNet) performance

### ISSUE THREAD

**Issue by seominseok0429** [wasidennis/AdaptSegNet#81]
I want to reproduce the baseline (ResNet) performance of table 1 in your paper(AdaptSegNet), can you share the same parameters as iteration? please.

**Comment by adricarda** [comment]
Were you able to reproduce the baseline results? I tried with the same hyper-parameters indicated in the supplementary material but sill no luck. I can reach at most 25% in terms of mIoU on the Cityscapes validation set.
Can you share the weights maybe?
thanks.

**Comment by chinacui** [comment]
@adricarda Same question. Have you solved it successfully? I have trained the Seg model source only and got ~24-25 mIoU accuracy. Thanks.

**Comment by seominseok0429** [comment]
> Were you able to reproduce the baseline results? I tried with the same hyper-parameters indicated in the supplementary material but sill no luck. I can reach at most 25% in terms of mIoU on the Cityscapes validation set.
> Can you share the weights maybe?
> thanks.


I was able to reproduce performance at 1 gpu. However, when I changed to 4gpu, I could not reproduce the performance. The hyperparameters were all the same.

**Comment by seominseok0429** [comment]
> @adricarda Same question. Have you solved it successfully? I have trained the Seg model source only and got ~24-25 mIoU accuracy. Thanks.

The hyperparameters were all the same. But we run 12 epochs. It is like 30000 iterations.

**Comment by wasidennis** [comment]
Please make sure that the pre-trained weight is successfully loaded (it also happened to us that the performance could drop to 2x if not doing so). Normally, the model should have 3x IoU in 10k iterations.

**Comment by seominseok0429** [comment]
I'll share our code. (soon)

**Comment by VE-yyq** [comment]
> I'll share our code. (soon)

I have the same question for reproducing. Is there anyone who can share the weights maybe?thanks.
